### ISSUE TITLE
[Question] Can this Application help human to just read topic he is interested in a wide forum / group?

### ISSUE THREAD

**Issue by pangtongs** [VincentTatan/EduTechAutoGenFaq#3]
Will this be helping me in being able to only see topic interested in within a group.

Imagine a case:
I am in a Whatsapp group with all my high school friends. 
They like to talk a lot about all topics and I am a passive participant in the group.
I am interested in `blockchain` and only interested in it.

With this kind or similar applications:
1. I can get the list of trending topics in this week.
2. If I found out that blockchain is one of trending topic and I can only look at that topic and only shown chats related to that topic

**Comment by VincentTatan** [comment]
Yes; this can be adapted to it. You could pull my code and replace the extraction data set with your preferred medium API (slack,whatsapp). You could use the same topic extractions and visualization to derive the same benefit for your usecases.
